File: C++/D101.cpp
#include <iostream>
using namespace std;

int main() {
    string Pnum;
    cin >> Pnum;
    char FirstDigit = Pnum[0];
    if (FirstDigit == '2' || FirstDigit == '3')
        cout << "Fixed" << endl;
    else
        cout << "Mobile" << endl;
    return 0;
}

File: Java/D101.java
import java.util.Scanner;

public class program {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        if (scan.hasNext()) {
            String a = scan.next();
            if ( a.charAt(0) == '2' || a.charAt(0) == '3') {
                System.out.println("Fixed");
            } else {
                System.out.println("Mobile");
            }
        }
    }
}
